### ISSUE TITLE
RecycleScroller after data update doesn't show new items on scroll

### ISSUE THREAD

**Issue by lukaszkups** [Akryum/vue-virtual-scroller#249]
Scenario: 

I use a `recycle-scroller` component for a table that receives some live data updates over the websocket.

On initial run, before any update come over the websocket, everything works fine.

When first update comes in, I see the data update (e.g. by console logging it), but table doesn't load any data when I scroll up / down (so I see huge blank spaces besides the part that was visible at the time when update come to me).

When the another update comes in though, it updates/renders/loads missing table rows in the place I was scrolled to in the moment when the update comes in.

The only workaround for this I've found is to toggle visibility of the whole `recycle-scroller` component on each data update, but it leads to scrolling user to the top on every update and table flickering (as the toggle must happen with `$nextTick` method to work so it's invisible for a split of a second).

Any suggestions will be highly appreciated.

**Comment by Akryum** [comment]
Is your data fully reactive? Do you have a reproduction that I can run?

**Comment by lukaszkups** [comment]
yes if I'm scrolled in the place where the update happened I see the updated row. Also I console log the amount of total rows which is also fine. 

I just found another workaround for this, by subscribing to `@scroll.native.passive` event and doing a `$forceUpdate` if it happens. 

I know this is a pain, but unfortunately I don't have reproduction code since I work on bigger table-like component that uses `recycle-scroller` for the data part inside, sorry. But I'll try to create a smaller reproduction code example soon.

**Comment by Akryum** [comment]
> yes if I'm scrolled in the place where the update happened I see the updated row. Also I console log the amount of total rows which is also fine.

It doesn't mean it's reactive. What if you `console.log` a watch over the array?

**Comment by lukaszkups** [comment]
just checked - it logs on every data update just fine

**Comment by lukaszkups** [comment]
I know that there's an event that is sent on each update ( `:emitUpdate='true'` etc.), but is there a way to actually SEND the event TO the `dynamic-scroller` component to force update it? (without need using `$forceUpdate` ?)

**Comment by Akryum** [comment]
It should totally be updating by itself, since it's basically watching the array.

**Comment by lukaszkups** [comment]
no no I mean to check if it fills the visible area with the table entries - because after I get the update and try to scroll, here's the example result: 

![image](https://user-images.githubusercontent.com/17531363/60353910-54b3ef80-99cb-11e9-8ec1-feb2c8b00cea.png)

(that white area is what I see when I scroll down after the update - before the update it draws new records just fine)

ahh I really need to get going with that code example :)

**Comment by mcfarljw** [comment]
This might be related to what I'm seeing in #256 when trying to change the array of item objects.
